blakserv/ccode.h

```diff
@@ -245,7 +245,10 @@ blak_int C_MinigameStringToNumber(int object_id,local_var_type *local_vars,
 		  int num_normal_parms,parm_node normal_parm_array[],
 		  int num_name_parms,parm_node name_parm_array[]);
 
-
+blak_int C_PointInSector(int object_id,local_var_type *local_vars,

```

Comment:
I wonder if maybe "IsPointInSector" is a better name?  "PointInSector" suggests that perhaps it finds and returns a point in a sector.